alexandria/settings.py

```diff
@@ -143,7 +153,7 @@ def parse_languages(languages):
     "DEFAULT_AUTHENTICATION_CLASSES": (
         "mozilla_django_oidc.contrib.drf.OIDCAuthentication",
     ),
-    "DEFAULT_PERMISSION_CLASSES": ("rest_framework.permissions.IsAuthenticated",),
+    "DEFAULT_PERMISSION_CLASSES": ("rest_framework.permissions.AllowAny",),
```

Comment:
This is just a temporary change until the permissions are implmented.

Reply:
In this case, please add a `# TODO ` comment in the code :)

Reply:
Done